tests/acceptance/bootstrap/FeatureContext.php

```diff
@@ -412,11 +412,15 @@ public function iExecuteArtisan($command)
 	 */
 	public function iShouldHaveCreatedOneSnapshotForEachSprint()
 	{
-		$numberOfSprints = Sprint::count();
+		$numberOfActiveSprints = count( array_filter( Sprint::all()->all(), function($sprint)
```

Comment:
Yeah, I've just subconsciously stuck to the style used in other projects. Fixed this